t/rose-config-diff/00-basic.t

```diff
@@ -305,7 +305,7 @@ file_cmp "$TEST_KEY.out" "$TEST_KEY.out" <<'__DIFF__'
     13	# description=Environment variable configuration
     14	[env]
     15	GEARSTICK_DECORATION=golfball
-    16	
+    16
```

Comment:
Will need to check these file comparisons still pass with whitespace stripped.

Reply:
Nope, the whitespace is functional.

Reply:
D'oh

Reply:
I've gone through every file where more than 1 line was changed and undone the whitespace changes in heredocs

Reply:
Thanks.